awscli/customizations/codedeploy/push.py

```diff
@@ -22,7 +22,7 @@
 from awscli.customizations.codedeploy.utils import validate_s3_location
 from awscli.customizations.commands import BasicCommand
 from awscli.errorhandler import ServerError, ClientError
-
```

Comment:
pep8: make sure there are two lines separating imports from where the code logic begins